index.html

```diff
@@ -5,9 +5,63 @@
 
     <link href="styles.css" rel="stylesheet" />
 
-    <title>Rocket Academy</title>
+    <title>Marina Zaitseva</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio page!</h1>
+    </section>
+    <img
+      id="avatar"
+      src="/Users/mz/Documents/bootcamp/m1/day3/portfolio-page/portfolio-page-bootcamp/avatar.JPG"
+      alt="avatar"
+    />
+    <section id="bio">
+      <h2>I am Marina</h2>
+      <h3>I am currently studying software engeneering at Rocket Academy</h3>
+    </section>
+    <section id="projects">
+      <div class="header">Here are my projects</div>
+      <div class="project-container">
+        <div class="project-tile">
+          <img
+            src="/Users/mz/Documents/bootcamp/m1/day3/portfolio-page/portfolio-page-bootcamp/blackjack.jpeg"
+          />
+          <div class="project-name">
```

Comment:
If it is just the project-name, maybe a `<span>` or a `<p>` would make more sense. subject though